cmd/dep/testdata/harness_tests/ensure/empty/case1/final/Gopkg.toml

```diff
@@ -27,12 +28,12 @@
 ## Optional: an alternate location (URL or import path) for the project's source.
 # source = "https://github.com/myfork/package.git"
 
-## Overrides have the same structure as [[dependencies]], but supercede all
-## [[dependencies]] declarations from all projects. Only the current project's
-## [[overrides]] are applied.
+## Override have the same structure as [[constraint]], but supercede all
```

Comment:
"Override" should probably be "Overrides" here to be grammatically correct and consistent with line 15.

Or, this sentence could be "Override has ..." and line 15 can be changed to "Constraint is a rule ..."